tox.ini

```diff
@@ -110,6 +126,16 @@ deps = -r{toxinidir}/lint-requirements.txt
 skip_install = true
 usedevelop = false
 
+[testenv:coverage]
+description = Generate HTML coverage report
+deps =
+    -r{toxinidir}/agent/requirements.txt
```

Comment:
Shouldn't we have all the agent and server requirement files loaded?  For example:

```
deps =
    -r{toxinidir}/agent/requirements.txt
    -r{toxinidir}/agent/test-requirements.txt
    -r{toxinidir}/server/requirements.txt
    -r{toxinidir}/server/test-requirements.txt
```